pkg/api/v1/routes/handlers.go

```diff
@@ -290,6 +287,21 @@ func (r *Routes) serverConditionCreate(c *gin.Context) (int, *v1types.ServerResp
 		metrics.PublishErrors.With(
 			prometheus.Labels{"conditionKind": string(condition.Kind)},
 		).Inc()
+
+		deleteErr := r.repository.Delete(otelCtx, serverID, condition.Kind)
+		if deleteErr != nil {
+			r.logger.WithFields(logrus.Fields{
+				"error": deleteErr,
+			}).Info("condition deletion failed")
```

Comment:
Nit: this should be a `Warn` so that it doesn’t get lost if someone filters logs.

Reply:
For some reason all of the error logging is `Info` level, I'd like to look into that separately. Will leave this as is for now.